### PR TITLE
feat(performance): removing tracers every hour

### PR DESCRIPTION
**Pull request by tvaintrob**

### CODE REVIEW

File: src/resource_monitor.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**
+ * @fileoverview monitor and manage epsagon resource usage
+ */
+
+const schedule = require('node-schedule');
+const { utils } = require('epsagon');
+const traceContext = require('./trace_context');
+
+const MAX_TRACERS = parseInt(process.env.EPSAGON_RESOURCE_MAX_TRACERS || '50', 10);
```
Comment (haimrait) [inline src/resource_monitor.js:9]: maybe change the default to be 0?

Comment (tvaintrob) [reply src/resource_monitor.js:9]: @galbash what do you think?

Comment (galbash) [reply src/resource_monitor.js:9]: I would start with 50 - just cause hopefully we won't leak in most cases, worst case we can change it before sending them

**Review by enoodle — APPROVED** [review]

**Pull request merged**